src/components/avatar.tsx

```diff
@@ -62,3 +64,20 @@ export const Avatar = (props: AvatarProps) => {
     </MjmlRaw>
   );
 };
+
+export const CivicLiker = (props: AvatarProps) => {
```

Comment:
Why do u need another component for CivicLiker Icon？

Reply:
true. removed!